File: Actividad7.c
/*
Nombre: Samara Mardueño Bernardino
Codigo: 222727303
Descripcion: Serie de fibonacci con validacion de numeros
*/

#include<stdio.h>
int main(){
    
    int i,repeticion,a = 1,b = 0,c = 0;
    printf("Dime cuantas veces quieres hacerlo: ");
    if(scanf("%d",&repeticion)== 1){
                 printf("Es numero\n"); 
                 repeticion+1;
                              
    
    
    for(i = 0; i < repeticion; i++){
          
          printf("El numero de fibonacci es: %d \n",b);
          
          c = a + b;
          a = b;
          b = c;
          
    }
               }else{
                     printf("No es numero");
                     }
    
    
    getch();
    return 0;
    }

File: Actividad7.py
#Nombre: Samara Mardueño Bernardino
#Codigo: 222727303
#Descripcion: Seria de fibonacci en python con valoracion


try:
    repeticion = int(input("Dime cuantas veces quieres hacerlo: "))
    
    a, b = 1, 0  # Valores iniciales
    for i in range(repeticion):
        print(f"El número de fibonacci es: {b}")
        c = a + b
        a, b = b, c

except ValueError:
    print("No es número")